app/code/local/LoyaltyLion/CouponImport/Model/Api2/Coupon/Rest/Admin/V1.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/* LoyaltyLion Coupon Import API 
+*
+* @category LoyaltyLion
+* @package LoyaltyLion_CouponImport
+* @author Patrick Molgaard 
+*/
+class LoyaltyLion_CouponImport_Model_Api2_Coupon_Rest_Admin_V1 extends LoyaltyLion_CouponImport_Model_Api2_Coupon
+{
+    /**
+     * Import LoyaltyLion coupons
+     *
+     * @param array $coupons
+     * @return string|void
+     */
+    protected function _multiCreate($coupons)
+    {
+        $ruleId = $this->getRequest()->getParam('rule_id');
+        $rule = $this->_loadSalesRule($ruleId);
+
+        $coupon = Mage::getModel('salesrule/coupon');
+        $now = $rule->getResource()->formatDate(
+            Mage::getSingleton('core/date')->gmtTimestamp()
+        );
+	//TODO: infer from parent rule
```

Comment:
Actually I still haven't worked out if it actually does anything.

It seems you can set an expiry both on individual coupons and on the price rule which contains them -- but the one on the coupon is never visible anywhere.

My gut feeling is that it isn't used anywhere but I'll do this just in case.